### ISSUE TITLE
version of fastai

### ISSUE THREAD

**Issue by WenjieWWJ** [IvLabs/stagewise-knowledge-distillation#14]
Hi,
Excellent work. May I know your version of fastai?
I had some import issues in the data pre-processing.

**Comment by akshaykvnit** [comment]
Hi, we hadn't installed any specific version. But, I checked the fastai version in our old conda environment and it was 1.0.60. So, you can try this. In general, I think fastai v1.0 should work.

**Comment by WenjieWWJ** [comment]
Got it. Thank you very much. Yes, I tried fastai v2.0. Some function names have been changed in v2.0. I also tried v1.0.40 and it worked.
Besides, may I know the PyTorch version in your old conda environment. It also has some compatibility problems.

The performance of stageKD is amazing and the code quality is very high. It would be better to release the version info in environment.yml, which helps to follow your work because fastai and torch changes a lot in later versions.
